styles/accessibility.css

```diff
@@ -0,0 +1,5 @@
+:focus-visible {
+  outline: 2px solid white!important; /* Outer color */
```

Comment:
Can we do this on `.nav-link:focus-visible, :focus-visible`  and drop the `!important` as otherwise this becomes challenging to override in local apps.

Reply:
done